dataplane/globalbase.cpp

```diff
@@ -1685,6 +1685,7 @@ generation::ServiceWeights(const balancer_service_t* service)
 
 		weights.emplace_back(real_id, weight);
 	}
+	return weights;
 }
```

Comment:
yea, like this, this should be just fixup-ed with the commit that adds `generation::ServiceWeights`